### DISCUSSION TITLE
Can I send code from Webots to the Robot?

### DISCUSSION THREAD

**Discussion by thephet** [cyberbotics/webots#5762] [General]
Hi there,

Sorry for this sort of newbie question. I have an Epson SCARA T6 robot (the real one) and I see you have it also in webots.

Can I write some controller in webots and then send it to the robot? So far the only way I know of controlling this robot is using their Epson RC+  software.

**Comment by omichel** [comment]
Webots won't allow you to transfer a control program to the real robot out-of-the-box.
However, this is certainly doable but you will have to implement a wrapper between the Webots API and the real robot API, e.g., the RC+ software or some other software interface.
[Cyberbotics](https://cyberbotics.com) has been developing this kind of interface for some customers in the past for their specific robots.
